packages/kbn-test/src/jest/utils/router_helpers.tsx

```diff
@@ -8,18 +8,39 @@
 
 import React, { Component, ComponentType } from 'react';
 import { MemoryRouter, Route, withRouter } from 'react-router-dom';
-import * as H from 'history';
+import { History, LocationDescriptor } from 'history';
```

Comment:
`LocationDescriptor` is defined at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/history/index.d.ts#L44